src/messages/table-empty.messages.js

```diff
@@ -0,0 +1,10 @@
+const { defineMessages } = require('react-intl');
+
+const tableEmptyMessages = defineMessages({
+  noRecords: {
+    id: 'tableEmpty.noRecords',
+    defaultMessage: 'No records'
```

Comment:
There are some filtering messages for empty states in this file `filtering.messages.js`. We should not introduce new strings. We have a lot of near-duplicates already.